presto-mongodb/src/main/java/com/facebook/presto/mongodb/MongoPageSource.java

```diff
@@ -164,7 +166,7 @@ else if (type.equals(DATE)) {
                     type.writeLong(output, TimeUnit.MILLISECONDS.toDays(utcMillis));
                 }
                 else if (type.equals(TIME)) {
-                    type.writeLong(output, UTC_CHRONOLOGY.millisOfDay().get(((Date) value).getTime()));
+                    type.writeLong(output, ZonedDateTime.ofInstant(((Date) value).toInstant(), UTC_ZONE_ID).get(ChronoField.MILLI_OF_DAY));
```

Comment:
Can we directly create an instant from the long instead of casting to a date and then converting it to an instant?   e.g. Instant.ofEpochMilli(value)

Reply:
![Screen Shot 2020-12-16 at 1 56 30 PM](https://user-images.githubusercontent.com/708608/102411255-82696b80-3fa6-11eb-8f6c-1fa2a41aefda.png)

Reply:
makes sense.  Thanks